.circleci/config.yml

```diff
@@ -0,0 +1,29 @@
+version: 2
```

Comment:
Looked into this and realized CircleCI requires this upgrade. Thanks again for doing this.